gcsio/src/main/java/com/google/cloud/hadoop/gcsio/ForwardingGoogleCloudStorage.java

```diff
@@ -63,15 +63,15 @@ public WritableByteChannel create(StorageResourceId resourceId, CreateObjectOpti
   }
 
   @Override
-  public void create(String bucketName) throws IOException {
+  public void createBucket(String bucketName) throws IOException {
     logger.atFine().log("%s.create(%s)", delegateClassName, bucketName);
```

Comment:
Should we update the logging string as well? Here and possibly anywhere else.

Reply:
Done.